Src/PerformanceCollector/Shared/Implementation/PerformanceCollectorEventSource.cs

```diff
@@ -154,6 +154,14 @@ public void TroubleshootingMessageEvent(string message, string applicationName =
             this.WriteEvent(14, message, this.ApplicationName);
         }
 
+        [Event(16, Level = EventLevel.Informational, Message = @"Performance counter is not available in the web app supported list. Counter is {0}.")]
```

Comment:
1. Make it error
2. Make is user actionable
3. Review the sentence with Michael and add more information on the impact of it.